### ISSUE TITLE
@gflops UndefVarError

### ISSUE THREAD

**Issue by ederag** [triscale-innov/GFlops.jl#32]
```julia
julia> x = 0.5; coeffs = rand(10);

julia> cnt = @count_ops evalpoly($x, $coeffs)
Flop Counter: 18 flop
┌────────┬─────────┐
│        │ Float64 │
├────────┼─────────┤
│ muladd │       9 │
└────────┴─────────┘

julia> @gflops evalpoly($x, $coeffs);
ERROR: UndefVarError: x not defined
Stacktrace:
 [1] macro expansion
   @ ~/.julia/packages/BenchmarkTools/ms0Xc/src/execution.jl:440 [inlined]
 [2] top-level scope
   @ ~/.julia/packages/GFlops/16nkd/src/count_ops.jl:44

julia> x
0.5
```
Julia Version 1.6.0
  [2ea8233c] GFlops v0.1.4

**Comment by ffevotte** [comment]
Thanks!

I can reproduce this with Julia 1.6.0 but not 1.5.4. Is that the case for you as well?

I just looked at the issue real quick, and it looks like it's not trivial. I'll try and understand a bit more what's going on...

**Comment by ffevotte** [comment]
Looks like this is fixed by #33:

```julia
julia> using GFlops
[ Info: Precompiling GFlops [2ea8233c-34d4-5acc-88b4-02f326385bcc]

julia> x = 0.5; coeffs = rand(10);

julia> cnt = @count_ops evalpoly($x, $coeffs)
Flop Counter: 18 flop
┌────────┬─────────┐
│        │ Float64 │
├────────┼─────────┤
│ muladd │       9 │
└────────┴─────────┘

julia> @gflops evalpoly($x, $coeffs);
  1.60 GFlops,  7.67% peak  (1.80e+01 flop, 1.13e-08 s, 0 alloc: 0 bytes)
```

**Comment by ederag** [comment]
Confirmed, it works on both `julia-1.5.4` and `julia-1.6.0` now, thanks !
